src/test/java/hw2/ex1/HomePageTest.java

```diff
@@ -0,0 +1,116 @@
+package hw2.ex1;
+
+import hw2.BaseClassTest;
+import java.util.ArrayList;
+import java.util.List;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+public class HomePageTest extends BaseClassTest {
+
+    private static final List<String> HEADER_SECTION_ITEMS_NAMES = new ArrayList<>(
+            List.of("HOME", "CONTACT FORM", "SERVICE", "METALS & COLORS"));
+
+    private static final int NUMBER_OF_IMAGES_AND_TEXTS = 4;
+
+    private static final List<String> ITEMS_IN_LEFT_SECTION = new ArrayList<>(
+            List.of("Home", "Contact form", "Service", "Metals & Colors",
+                    "Elements packs")
+    );
+
+    private static final List<String> TEXTS_UNDER_IMAGES = new ArrayList<>(
+        List.of(
+            "To include good practices\n"
+                + "and ideas from successful\n"
+                + "EPAM project",
+
+            "To be flexible and\n"
+                + "customizable",
+
+            "To be multiplatform",
+
+            "Already have good base\n"
+                + "(about 20 internal and\n"
+                + "some external projects),\n"
+                + "wish to get more…"
+        )
+    );
+
+    @Test
+    public void checkElementsTest() {
+        //1. Open test site by URL
+        webDriver.navigate().to(BASE_URL);
+
+        //2. Assert Browser title
+        softAssertions.assertThat(webDriver.getTitle()).isEqualTo(BROWSER_TITLE);
+
+        //3. Perform login
+        webDriver.findElement(By.id("user-icon")).click();
+        webDriver.findElement(By.id("name")).sendKeys(USERNAME);
+        webDriver.findElement(By.id("password")).sendKeys(PASS);
+        webDriver.findElement(By.id("login-button")).click();
+
+        //4. Assert Username is loggined
+        WebElement user = webDriver.findElement(By.id("user-name"));
+        softAssertions.assertThat(user.isDisplayed()).isTrue();
```

Comment:
Why it's decided to use soft assertions for the whole test? Soft assertions doesn't stop test execution but sometimes it leads to unexpected errors and exceptions which really hard to analyze. I would recommend to use soft assertions only when it is needed (in loops, streams, when verifying tables data, xml files, responses from DB...)
Try to provide incorrect username or password and verify where will test fail.

Reply:
I was pretty sure the task was to use only soft assertions, but I can't find it right now.  In case of use soft assertion to check incorrect login/password - you can determine where the test fell. (You can see it on screenshot). But anyway I will refactor code in this HW and others to use not only soft assertions. 
![image](https://user-images.githubusercontent.com/74490705/141360353-d58ad30d-5018-4d81-8c31-bb0cb44a00f5.png)